org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/node/NodeRunDAPDebugDelegate.java

```diff
@@ -316,7 +320,11 @@ public Map<String, Object> readJSonFile(File tsConfgFile) {
 				response.append(inputLine).append('\n');
 			}
 			Type type = new TypeToken<Map<String, Object>>() {}.getType();
-			return new Gson().fromJson(response.toString(), type);
+			var responseString = response.toString();
+			responseString = BlockCommentPattern.matcher(responseString).replaceAll("");
+			responseString = LineCommentPattern.matcher(responseString).replaceAll("");
+			responseString = TrailingCommaPattern.matcher(responseString).replaceAll("$1}");
```

Comment:
Can you please extract it into a dedicated method with a nice name and some documentation to not forget what case they're serving?

Reply:
Done by acdd258.  The docstring doesn't make explicit mention of what kind of preprocessing is done, so that if any other JSON tweaks are needed, the documentation & method name may remain unchanged & continue to be accurate.